test/controllers/overseas.name.controller.spec.ts

```diff
@@ -0,0 +1,140 @@
+jest.mock("ioredis");
+jest.mock('../../src/middleware/authentication.middleware');
+jest.mock('../../src/utils/application.data');
+jest.mock('../../src/middleware/navigation/is.secure.register.middleware');
+jest.mock('../../src/service/transaction.service');
+jest.mock('../../src/service/overseas.entities.service');
```

Comment:
do you need to mock the service.availibility.middleware? it might be relying on the test setup process env defaults, just thinking might be better to mock it.

Reply:
As you said we have already `SHOW_SERVICE_OFFLINE_PAGE` set to false on test setup and the `FEATURE_FLAG_ENABLE_ROE_UPDATE`  is not part of our journey. I think is better not to mock it